accession/models.py

```diff
@@ -107,7 +114,10 @@ class Donor(models.Model):
     middle_name = models.CharField(blank=True, max_length=100)
     last_name = models.CharField(blank=True, max_length=100)
     organization_name = models.CharField(blank=True, max_length=255)
-    donor_type = models.CharField(max_length=3, blank=True, choices=DONOR_TYPE_CHOICES, default="PER")
+    donor_type = models.CharField(max_length=3,
+                                  blank=True,
+                                  choices=DONOR_TYPE_CHOICES,
+                                  default="PER")
```

Comment:
This formatting is different from all the others. This is not wrong, nor do you need to change it. I just wanted to make sure you are aware of that.